tests/yam/agama/agama.pm

```diff
@@ -1,33 +1,38 @@
-## Copyright 2023 SUSE LLC
+## Copyright 2024 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 # Summary: Run interactive installation with Agama,
-# run playwright tests directly from the Live ISO.
+# using a web automation tool to test directly from the Live ISO.
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
 use base Yam::Agama::agama_base;
 use strict;
 use warnings;
+use testapi;
 
 use testapi qw(
   assert_script_run
   assert_screen
   get_required_var
   enter_cmd
+  select_console
 );
 
 sub run {
     my $self = shift;
     my $test = get_required_var('AGAMA_TEST');
 
-    assert_script_run("playwright test --trace on --project chromium --config /usr/share/e2e-agama-playwright tests/" . $test . ".spec.ts", timeout => 1200);
-    $self->upload_traces();
-    enter_cmd 'reboot';
+    assert_script_run("agama-integration-tests /usr/share/agama/integration-tests/build/" . $test . ".js", timeout => 1200);
+
+    select_console 'displaymanager';
```

Comment:
I never seen use this console, where did you get this? I think the installation console should be fine with a simple needle, like covering the button, for this case for display manager the needle you created (or was created already) is too much, needs to be smaller, because the screen matching will be very slow, given that this needle is very generic.

Reply:
I used this one because I see that the majority of UI screens are stored at this section but, of course I can swap to another one.